internal/model/manifest.go

```diff
@@ -192,14 +192,17 @@ type Run struct {
 	// Required. The command to run.
 	Cmd Cmd
 	// Optional. If not specified, this command runs on every change.
-	// If specified, we only run the Cmd if the trigger matches the changed file.
-	Triggers []string
-	// Directory the Triggers are relative to
-	BaseDirectory string
+	// If specified, we only run the Cmd if the changed file matches a trigger.
+	Triggers PathSet
 }
 
-func (r Run) WithTriggers(triggers []string) Run {
-	r.Triggers = triggers
+func (r Run) WithTriggers(paths []string, baseDir string) Run {
+	if len(paths) > 0 {
```

Comment:
i don't think this check is right? if len(paths) == 0, it should write an empty PathSet, instead of keeping the previous value.

Reply:
fair! (I wanted to avoid a pathset with empty Paths and populated baseDir, cuz that seems weird to me, but 🤷‍♀️ )